.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,33 @@
+# This workflows will upload a Python Package using flit when a release is created
```

Comment:
Looks like we don't use `flit` but use `setuptools` and `twine` ?

Reply:
Ah, holdover from converting the script...

Reply:
(I started with flit, converted to setuptools for some projects that use setuptools, so I know it works)

Reply:
```suggestion
# This workflows will upload a Python Package using setup.py/twine when a release is created
```